### PR TITLE
Ide 3972 description

### PR DESCRIPTION
**Pull request by yanan**

### CODE REVIEW

**Comment by jtydhr88** [comment]
SF Successfully on liferay-ide folder

File: tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/tasks/FindUpgradeProblemsTask.java [hunk 1]
```diff
@@ -24,7 +24,10 @@
  * @author Terry Jia
  */
 @Component(
-	property = {"categoryId=code", "id=find_upgrade_problems", "order=5", "title=Find Upgrade Problems"},
+	property = {
+		"categoryId=code", "id=find_upgrade_problems", "order=5", "title=Find Upgrade Problems",
+		"description=In this section, it finds affected plugin code and shows documentation that describes how to adapt it, helps to address the problems, to auto-correct some problems. It facilitates opening affected code in an editor to check and modify it."
```
Comment (jtydhr88) [inline tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/tasks/FindUpgradeProblemsTask.java]: is that ok for sf?

Comment (yanan) [reply tools/plugins/com.liferay.ide.upgrade.problems.core/src/com/liferay/ide/upgrade/problems/core/internal/tasks/FindUpgradeProblemsTask.java]: Running sf on liferay-ide root folder, sf successfully

**Comment by yanan** [comment]
Rebased to master and resend to Greg in pr 
https://github.com/gamerson/liferay-ide/pull/4063

**Pull request closed without merging**